<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelMatcherLogEnable</class>
 <widget class="QFrame" name="CModelMatcherLogEnable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="hl_ModelMatcherLogEnable">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_Enable">
     <property name="toolTip">
      <string>Enables / disables future message logging</string>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_LogReverseLookup">
     <property name="toolTip">
      <string>enable logging of reverse lookup messages</string>
     </property>
     <property name="text">
      <string>rev. lookup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_LogMatchingMessages">
     <property name="toolTip">
      <string>enable matching messages</string>
     </property>
     <property name="text">
      <string>matching</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_LogDetailed">
     <property name="toolTip">
      <string>a simplified matching log (faster) or a detailed log (slower)</string>
     </property>
     <property name="text">
      <string>detailed</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="hs_ModelMatcherLogEnable">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_LogReverseLookup</tabstop>
  <tabstop>cb_LogMatchingMessages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
